build_info/bsdmake.control

```diff
@@ -0,0 +1,8 @@
+Package: bsdmake
```

Comment:
Debianize this 
https://packages.debian.org/sid/bmake

Reply:
These are different packages, this is `bsdmake`, that is `bmake`, they are different implementations of BSD make.